### DISCUSSION TITLE
Where is the cached stored?

### DISCUSSION THREAD

**Discussion by jcguu95** [minad/consult#965] [Q&A] [answered]
Where does `consult` store its cache? And how do I ask it to save cache automatically?
Sometimes I have to kill my emacs without grace, which wipes out consult cache.

**Comment by minad** [comment, marked as answer]
I am not entirely sure what you mean by cache, maybe the minibuffer history? You can enable `savehist-mode` to save the history across Emacs restarts as is recommended for example in the [Vertico README](https://github.com/minad/vertico?tab=readme-ov-file#configuration).
